packages/next/package.json

```diff
@@ -69,7 +69,7 @@
     "http-status": "1.0.1",
     "launch-editor": "2.2.1",
     "loader-utils": "1.1.0",
-    "mkdirp-then": "1.2.0",
+    "mkdirp": "0.5.1",
```

Comment:
I'd go with `make-dir` since it's already in the dependency tree via `find-cache-dir`.